No webui's

Is it expected that webui's which have been split of recently cannot be installed anymore in the snapshots?

CC: @davidgraeff

@maggu2810 if you would know by heart what I need to get this dependency in the correct way:

```
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-compiler-plugin:3.8.0:compile (default-compile) on project org.openhab.ui.classic: Compilation failure
[ERROR] D:\Documents\_Martin\Java\openhab-webui\org.openhab.ui.classic\src\main\java\org\openhab\ui\classic\internal\render\PageRenderer.java: The type org.eclipse.emf.common.util.EList cannot be resolved. It is indirectly referenced from required .class files

After the manual install, only these problems remain:

> 2019-02-22 13:43:59.376 [ERROR] [core.karaf.internal.FeatureInstaller] - Failed installing 'openhab-misc-lsp, openhab-misc-dropbox1, openhab-binding-yahooweather, openhab-misc-restdocs'

--- a/distributions/openhab/src/main/filtered-resources/userdata/etc/org.apache.karaf.features.cfg
+++ b/distributions/openhab/src/main/filtered-resources/userdata/etc/org.apache.karaf.features.cfg
@@ -42,6 +42,7 @@ featuresBoot = \
     service/${karaf.version}, \
     jaas/${karaf.version}, \
     openhab-runtime-base, \
+    openhab-ui-dashboard, \
     deployer/${karaf.version}, \
     diagnostic/${karaf.version}, \
     bundle/${karaf.version}, \

